pkgs/development/tools/pnpm/fetch-deps/default.nix

```
@@ -32,8 +33,14 @@
             outputHash = "";
             outputHashAlgo = "sha256";
           };
-      installFlags = lib.optionalString (pnpmWorkspace != "") "--filter=${pnpmWorkspace}";
+
+      filterFlags = lib.map (package: "--filter=${package}") pnpmWorkspaces;
     in
+    # pnpmWorkspace was deprecated, so throw if it's used.
+    assert (lib.throwIf (!args ? pnpmWorkspace)
```

We want to throw when a user DID give a `pnpmWorkspace`
```suggestion
    assert (lib.throwIf (args ? pnpmWorkspace)
```